test/Test/Time/Property.hs

```diff
@@ -41,10 +51,21 @@ unitChooser 8 t = MkAnyTime $ Time @WeekUnit        t
 unitChooser 9 t = MkAnyTime $ Time @FortnightUnit   t
 unitChooser _ _ = error "Impossible happened"
 
--- | Verifier for 'AnyTime'.
+-- | Verifier for 'AnyTime' @read . show = id@.
 verifyAnyTime :: (MonadTest m) => AnyTime -> m ()
 verifyAnyTime (MkAnyTime t) = read (show t) === t
 
+-- | Verifier for 'toUnit'.
+verifyToUnit :: forall m . (MonadTest m) => AnyTime -> AnyTime -> m ()
+verifyToUnit (MkAnyTime t1) (MkAnyTime t2) = asd t1 t2
+  where
+    asd :: forall (a :: Rat) (b :: Rat) .
+           (KnownSymbol (UnitName a), KnownRat a, KnownRat b)
+        => Time a
+        -> Time b
+        -> m ()
+    asd t _ = withRuntimeDivRat @b @a (toUnit @a (withRuntimeDivRat @a @b $ toUnit @b t)) === t
```

Comment:
I think it's better to rearrange functions applications here and move both `withRuntimeDivRat` into beginning. Is it possible? It makes property more readable.

Reply:
Also, I think it's enough to specify only type of `unitTo` once. Type inference will do the rest of the work. Just

```haskell
toUnit (toUnit @unitTo t) === t
```